src/components/typography/Label.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import { forwardRef, PropsWithChildren } from 'react';
+
+interface Props {
+  className?: string;
+}
+
+export const Label = forwardRef<HTMLDivElement, Readonly<PropsWithChildren<Props>>>(
+  ({ children, className, ...rest }, ref) => (
+    <StyledLabel {...{ children, className, ref, ...rest }} />
+  ),
+);
+
+Label.displayName = 'Label';
+
+export const StyledLabel = styled.div`
```

Comment:
Let's rename it `Label`, add the `displayName = 'Label'`, restrict the props to your Props interface, and make it a `label` instead of a `div` by default.